### PR TITLE
Spell length - Climatic Methods Menus

### PR DESCRIPTION
**Pull request by aduut**
Modifications to Spell length, adding all parameters, some parameters…
… including years are a vector of numbers which are currently not implemented.

### CODE REVIEW

**Comment by volloholic** [comment]
One small correction then I can merge

File: instat/dlgSpells.Designer.vb [hunk 1]
```diff
@@ -22,8 +22,196 @@ Partial Class dlgSpells
     'Do not modify it using the code editor.
     <System.Diagnostics.DebuggerStepThrough()> _
     Private Sub InitializeComponent()
-        components = New System.ComponentModel.Container
+        Me.chkPrintTable = New System.Windows.Forms.CheckBox()
+        Me.chkMonthAbbreviations = New System.Windows.Forms.CheckBox()
+        Me.chkRemoveNA = New System.Windows.Forms.CheckBox()
+        Me.chkSeparate = New System.Windows.Forms.CheckBox()
+        Me.Label1 = New System.Windows.Forms.Label()
+        Me.Threshold = New System.Windows.Forms.Label()
```
Comment (volloholic) [inline instat/dlgSpells.Designer.vb]: Should this be lblThreshold?

**Comment by volloholic** [comment]
Again I still can't merge until you correct and commit again.

**Comment by stevekogo** [comment]
Fixed the code in  dlgSpellLength

**Pull request closed without merging**